### DISCUSSION TITLE
Styrbar Remote control IKEA left and right event after long press missing

### DISCUSSION THREAD

**Discussion by luckyzor** [zachowj/node-red-contrib-home-assistant-websocket#882] [General]
I want to configure this remote for dim some bulbs and I need the release event.
Unfortunately, it doenst exists in the device configuration:
![image](https://user-images.githubusercontent.com/31155573/224322415-e3d0ca77-daef-4191-be4b-6024c3912ddc.png)

But in the zha_events we have the event after a long press,  left button event after long press button: 
![image](https://user-images.githubusercontent.com/31155573/224322636-17931e27-f19c-4710-9bb9-002f5f0ef311.png)

are there anyway to add this to the device configuration?

thanks
Ivan

**Comment by zachowj** [comment]
The device node is driven from data directly obtained from HA [device triggers](https://www.home-assistant.io/docs/automation/trigger/#device-triggers). If it doesn't exist, HA isn't supporting it for that device.

If your case you can use an events:all node with the event type set to `zha_event` and the event data 
```
{
  "device_id": "9206...", (fill the rest in)
  "command": "release"
}
```
